test/services/supplementary-billing/check-live-bill-run.service.test.js

```diff
@@ -24,7 +24,7 @@ describe('Check Live Bill Run service', () => {
   describe('when an sroc supplementary bill run exists for this region and financial year', () => {
     describe('with a status considered to be "live"', () => {
       beforeEach(async () => {
-        billRun = await BillingBatchHelper.add()
+        billRun = await BillingBatchHelper.add({ status: 'queued' })
```

Comment:
Going to comment here but also applies to line 39 in the same file and the change to `test/support/helpers/water/billing-batch.helper.js`.

`queued` is the default state for a new BillingBatch record and that is why the helper uses that as its default. Is there a reason why we have switched the default to `sent`?

Reply:
I defaulted it to `sent` as there was only 2 tests that used that default value `queued`. As the majority of the tests use the status of `sent` I found the helper to be much more helpful if this was the default.

Reply:
I think I'd still argue that the default matches with what would happen when the record is newly created and that we then apply the status we need in the tests.

It might be we have more tests that use 'sent' now. But that might change in the future. Plus, the status being `sent` is important in these tests whereas it might be meaningless in others. So, IMHO it's better to set the state where it matters. 🤷

Reply:
You've talked me round :grin: I shall change the default back to `queued`